doc/python/axes.md

```diff
@@ -559,6 +559,23 @@ fig.update_yaxes(range=[3, 9])
 fig.show()
 ```
 
+#### Exclude Inside Tick Labels from Range
+
+You can use `insiderange` instead of `range` on an axis if you have tick labels positioned on the inside of another axis and you don't want the range to overlap with those labels.
```

Comment:
```suggestion
#### Exclude Inside Tick Labels from Range

*New in 5.18*

You can use `insiderange` instead of `range` on an axis if you have tick labels positioned on the inside of another axis and you don't want the range to overlap with those labels.
```